client/src/helpers/APICalls/updateCard.ts

```diff
@@ -0,0 +1,17 @@
+import { IFetchOptions, ICard } from '../../interface';
+
+const updateCard = async (card: ICard): Promise<ICard> => {
```

Comment:
If all we're using is the Id why pass the whole card over?

Reply:
Update to pass only the card name and an object of parameters to change :)

Reply:
Ahh, yes! Sorry, I only seen the `card._id` 😂